docs/guide/models_and_layers.md

```diff
@@ -15,7 +15,7 @@ There are two ways to create a model using the Layers API: A _sequential_ model,
 
 ### The sequential model
 
-The most common type of model is the <code>[Sequential](https://js.tensorflow.org/api/0.15.1/#class:Sequential)</code> model, which is a linear stack of layers. You can create a <code>Sequential</code> model by passing a list of layers to the <code>[sequential()](https://js.tensorflow.org/api/0.15.1/#sequential)</code> function:
+The most common type of model is the <code>[Sequential](https://js.tensorflow.org/api/latest/#sequential)</code> model, which is a linear stack of layers. You can create a <code>Sequential</code> model by passing a list of layers to the <code>[sequential()](https://js.tensorflow.org/api/0.15.1/#sequential)</code> function:
```

Comment:
can you update the second link as well? thanks

Reply:
Hi @pyu10055 ,
I am very sorry that I didn't check the other broken sequential link. I have updated the link now. Please do the needful. Thank You!